src/freshdeskService.ts

```diff
@@ -163,3 +169,56 @@ export function generateFreshdeskTicketFieldDefinitions(
     return memo
   }, [])
 }
+
+/**
+ * This function is used to decide if `options` or `choices` should be used. If
+ * there is no `options` parameter passed in, it will use the `choices`
+ * configuration to create the options required for the options required.
+ *
+ * @param options
+ * @param choices
+ * @returns
+ */
+export function transformFreshdeskChoicesIntoOptions(
```

Comment:
Prob makes more sense to make this function take in `FreshdeskTypes.FreshdeskField[]` and return `FreshdeskTypes.FreshdeskField[]` and name the function `setFreshdeskFieldOptions()`. Makes it a bit clearer what is doing and how to call it.